alfresco-search/src/main/java/org/alfresco/solr/AlfrescoSolrDataModel.java

```diff
@@ -1411,7 +1411,7 @@ public Query getCMISQuery(CMISQueryMode mode, Pair<SearchParameters, Boolean> se
     public Solr4QueryParser getLuceneQueryParser(SearchParameters searchParameters, SolrQueryRequest req, FTSQueryParser.RerankPhase rerankPhase)
     {
         Analyzer analyzer =  req.getSchema().getQueryAnalyzer();
-        Solr4QueryParser parser = new Solr4QueryParser(req, Version.LUCENE_5_5_0, searchParameters.getDefaultFieldName(), analyzer, rerankPhase);
+        Solr4QueryParser parser = new Solr4QueryParser(req, Version.LATEST, searchParameters.getDefaultFieldName(), analyzer, rerankPhase);

```

Comment:
The only conflict was in this file. I resolved it by keeping the changes from master, but with this one change.

Reply:
That's correct, in 2.0.x we replaced the hard-coded version with Version.LATEST so the change is good